.ci/run.sh

```diff
@@ -83,7 +83,7 @@ case "${CI_JOB}" in
 		info "Running Confidential Containers tests for AMD SEV-SNP"
 		sudo -E PATH="$PATH" CRI_RUNTIME="containerd" bash -c "make cc-snp-kubernetes"
 		;;
-	"CC_CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD_K8S_TDX_QEMU"|"CC_CRI_CONTAINERD_K8S_SE_QEMU"|"CC_CRI_CONTAINERD_K8S_TDX_CLOUD_HYPERVISOR")
+	"CC_CRI_CONTAINERD_K8S"|"CC_CRI_CONTAINERD_K8S_TDX_QEMU"|"CC_CRI_CONTAINERD_K8S_SE_QEMU"|"CC_CRI_CONTAINERD_K8S_TDX_CLOUD_HYPERVISOR"|"CC_CRI_CONTAINERD_K8S_IMAGE_OFFLOAD_TO_GUEST")
```

Comment:
Do we still need the `CC_CRI_CONTAINERD_K8S_IMAGE_OFFLOAD_TO_GUEST` job if the 'original' ones are testing the image-offload to guest now?

Reply:
Oh - I've just noticed it was removed in a later commit, hence showing outdated